frontend/src/metabase/palette/components/PaletteResults.tsx

```diff
@@ -92,6 +92,7 @@ export const PaletteResults = () => {
                   <Flex gap=".5rem" style={{ minWidth: 0 }}>
                     {item.icon && (
                       <Icon
+                        aria-hidden
```

Comment:
So I adjust the accessibility tree so that `option` will have the correct label. Looking at the screenshot below
![image](https://github.com/metabase/metabase/assets/1937582/98fec8b4-bc74-4bd7-9852-d655a7733f16)

You can see that we can ignore the first icon since it's just a decorative element, and we could also remove `Open` plus its icon from the tree because this is already an option that users have to select anyway so `Open` serves only as a visual aid.